nodejs/extensions.bzl

```diff
@@ -2,21 +2,33 @@
 
 load(":repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")
 
+DEFAULT_NODE_REPOSITORY_NAME = "nodejs"
+
 def _toolchain_extension(module_ctx):
     registrations = {}
     for mod in module_ctx.modules:
         for toolchain in mod.tags.toolchain:
-            if toolchain.name in registrations.keys():
-                if toolchain.node_version == registrations[toolchain.name]:
+            if toolchain.name != DEFAULT_NODE_REPOSITORY_NAME and not mod.is_root:
+                fail("Only the root module may provide a name for the node toolchain.")
+
+            name = toolchain.name if toolchain.name else DEFAULT_NODE_REPOSITORY_NAME
```

Comment:
how about just setting the `default` of the `name` attribute below?

Reply:
Good point. Fixed.